.genx/templates/angular/entityManager.hbs

```diff
@@ -1,45 +1,53 @@
-<entity-management
-    design-system-prefix="rapid"
-    enable-row-flashing
-    enable-cell-flashing
-    {{#if config.title}}
-    title="{{ config.title }}"
-    {{/if}}
-    resourceName="{{ config.resourceName }}"
-    {{#if config.createEvent}}
-    createEvent="{{ config.createEvent }}"
-    {{#if config.createFormUiSchema}}
-    [createFormUiSchema]="tile{{ config.index }}.createFormUiSchema"
-    {{/if}}
-    {{/if}}
-    {{#if config.updateEvent}}
-    updateEvent="{{ config.updateEvent }}"
-    {{#if config.updateFormUiSchema}}
-    [updateFormUiSchema]="tile{{ config.index }}.updateFormUiSchema"
-    {{/if}}
-    {{/if}}
-    {{#if config.deleteEvent}}
-    deleteEvent="{{ config.deleteEvent }}"
-    {{/if}}
-    {{#if config.gridOptions}}
-    [gridOptions]="{{ config.gridOptions }}"
-    {{/if}}
-    {{#if config.snapshot}}
-    [datasourceConfig]="{ isSnapshot: {{ config.snapshot }} }"
-    {{/if}}
-    {{#if config.reqrep}}
-    [datasourceConfig]="tile{{ config.index }}.reqrep"
-    {{/if}}
-    {{#if config.columns}}
-    [columns]="tile{{ config.index }}.columns"
-    {{/if}}
-    {{#if config.modalPosition}}
-    modal-position="{{ config.modalPosition }}"
-    {{/if}}
-    {{#if config.sizeColumnsToFit}}
-    size-columns-to-fit
-    {{/if}}
-    {{#if config.enableSearchBar}}
-    enable-search-bar
-    {{/if}}
-></entity-management>
+<section style="contain: content; display: flex; height: 100%; width: 100%; overflow-y: auto; align-items: center; justify-content: center;">
+  <ng-container *ngIf="canUserViewElement('{{config.permissions.viewRight}}'); else notPermitted{{ config.index }}">
+    <entity-management
```

Comment:
After running the application locally, entity-management is not visible - I haven't looked into it, but adding the following fixes it:

```
    width: 100%;
    height: 100%;
```

Video:

https://github.com/genesiscommunitysuccess/blank-app-seed/assets/22753085/3995b325-bd73-4a54-a5a6-a5a61bda9112


 note: I haven't tested grid-pro - I don't know if the same thing happens

Reply:
css fixed